### PR TITLE
feat: add 4 scenario for training ANN

### PR DESCRIPTION
**Pull request by storyofhis**

### CODE REVIEW

**Comment by storyofhis** [comment]
nambahin activation function buat training dataset 
- relu
- tanh
- sigmoid 
- softmax
